### DISCUSSION TITLE
Suspended account and data recovery

### DISCUSSION THREAD

**Discussion by Este13** [ReVanced/revanced-discussions#1053] [Question]
It's my 3rd suspended account, i use the  revanced app on my rooted samsung and i have to change it every 4 to 6 months. Every time i asked google the reason, and never got an answer. I dont really care to lose my accounts, i can make others easily, my problem is that im a big YT user, and every time i loose everything : playlists, likes, subscriptions. 
The thing is it's impossible to make a Google Takeout on a suspended account. I now i should have done it before. 

The interesting thing, is that this time i cant connect to my account on any webpage, it will show a message telling me my account is suspended, but im still connected to te revanced app and have access and can use everything. Tell me if im wrong but i think revanced use a lot the cache and local data to limit the requests to Google. After some time i finally found and extracted my app.revanced.android.youtube folder that contains 719mo of data. 

Can i get some help on how to extract the data im looking for? Another method? (already tried youtube data api) 

And any idea on why my accounts get banned again and again?

**Comment by oSumAtrIX** [comment]
ReVanced is unrelated to your problem. Contact Google support

**Reply by Este13** [reply]
Why did you close the ticket? 
The reason im asking here i because google never responded to anything.
Im not asking to do something with my accounts, im asking help to extract data form the app files, so it is related.
